From 5955832084f8416a1b2f3691b2693829245cebd7 Mon Sep 17 00:00:00 2001
From: Steven Chen <cw@rock-chips.com>
Date: Wed, 12 Jul 2017 11:51:34 +0800
Subject: [PATCH] ARM: dts: rk3288-android: enable dmc auto frequency
 adjustment

Enable dmc auto frequency adjustment and
set dmc minimum frequency to 400MHz for better performance

Change-Id: I0d2a192f438c23d879158f55dc948205f67f8663
Signed-off-by: Steven Chen <cw@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-android.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/rk3288-android.dtsi b/arch/arm/boot/dts/rk3288-android.dtsi
index 37892c732e9a..28084c7baa32 100644
--- a/arch/arm/boot/dts/rk3288-android.dtsi
+++ b/arch/arm/boot/dts/rk3288-android.dtsi
@@ -84,8 +84,8 @@
 			SYS_STATUS_DUALVIEW	396000
 			SYS_STATUS_ISP		396000
 		>;
-		auto-min-freq = <192000>;
-		auto-freq-en = <0>;
+		auto-min-freq = <400000>;
+		auto-freq-en = <1>;
 		status = "diasbled";
 	};
 
-- 
2.35.3

